mount_darwin.go

```diff
@@ -90,12 +98,37 @@ func isBoringMountOSXFUSEError(err error) bool {
 	return false
 }
 
-func callMount(bin string, daemonVar string, dir string, conf *mountConfig, f *os.File, ready chan<- struct{}, errp *error) error {
+func receiveDeviceFD(ourSocketFD int) (*os.File, error) {
+	oob := make([]byte, syscall.CmsgLen(8)) // data is FD. Assume 8 bytes to be safe
```

Comment:
What does `oob` stand for?  I can only think of "out of bounds".

Reply:
I think it's out-of-bands. It's the name that `syscall` package uses. I guess I could link this in comments https://www.freebsd.org/cgi/man.cgi?query=recvmsg&sektion=2